app/controllers/activities_controller.rb

```diff
@@ -4,6 +4,25 @@
 class ActivitiesController < AuthenticatedController
   include ProjectScoped
 
+  def index
+    activities = Activity.includes(:trackable)
+                         .order(created_at: :desc)
+                         .page(params[:page])
+
+    @page = params[:page].present? ? params[:page] : 1
```

Comment:
Since we're initialising a `@page` variable anyway, let's do it before the previous call, that way you can pass it as argument to `page()`.

If this is a number, call `.to_i` on the param.

Reply:
Got it!